Articles/Blog/2023-03-Julia_for_BSSw.md

```diff
@@ -1,52 +1,60 @@
 
 **Compose to prevent inheritance bloat**
-Julia does not support object-oriented as C++ or Python. Julia projects are organized by modules and proper data locality and composition using "data container" structs and type hierarchy trees in which abstract types have no members, see related [discussion](https://github.com/JuliaLang/julia/issues/4935). Think of composition as Derived "has-a" Base, instead of Derived "is-a" Base. This weak-coupling prevents long hierarchies of classes that can go out of hand really quickly, while encouraging software developers to think of structs as data containers for which operations are applied - as one would do in languages like pre-2003 Fortran, R or C.
+Julia does not support object-oriented as C++ or Python. Julia projects are organized by modules and proper data locality and composition using "data container" structs and type hierarchy trees in which abstract types have no members (see related [discussions](https://github.com/JuliaLang/julia/issues/4935)). Think of composition as Derived "has-a" Base, instead of Derived "is-a" Base. This weak coupling prevents deep hierarchies of classes that can quickly get out of hand, while encouraging software developers to think of structs as data containers to which operations are applied - as one would do in languages like pre-2003 Fortran, R, or C.
 
 **Interoperate with existing software**
-Julia enables lightweight reusability of existing Fortran and C infrastructure via the [`@ccall` macro](https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/). Similarly, Python and R interoperability is possible with [PyCall.jl](https://github.com/JuliaPy/PyCall.jl) and [RCall.jl](https://github.com/JuliaInterop/RCall.jl), respectively. So Julia does not promote "reinventing", but rather "reusing" existing wheels (pun not intended) as mature scientific software can be a large investments.
+Julia enables lightweight reusability of existing Fortran and C infrastructure via the [`@ccall` macro](https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/). Similarly, Python and R interoperability is possible with [PyCall.jl](https://github.com/JuliaPy/PyCall.jl) and [RCall.jl](https://github.com/JuliaInterop/RCall.jl), respectively. So Julia promotes reuse over reinvention, which is both important and useful given the volume of mature scientific software.
+
+**Jupyter and Pluto.jl notebooks**
+Computational notebooks, Jupyter in particular, have been widely adopted in science and do not need an introduction. Jupyter is powered by Anaconda with Python kernels requiring setting a conda environment for managing the required dependencies before launching the server and web client interface. Notebooks are stored using a `*.ipynb` file format based on JSON. 
+
+*David: I don't understand this sentence.  Are we talking about Jupyter still or Pluto?  Should this sentence be in the following paragraph?*
```

Comment:
Good catch. I would rephrase it as: "Jupyter supports interactive Julia notebooks via the [IJulia](https://github.com/JuliaLang/IJulia.jl) kernel package that can understand information readily available..."